### PR TITLE
Adding zip code migrations and CSV file 

### PR DESCRIPTION
**Pull request by maneshkommidi**
https://github.com/department-of-veterans-affairs/gibct-data-service/issues/324

### CODE REVIEW

File: db/migrate/20180522142853_create_zipcode_rates.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class CreateZipcodeRates < ActiveRecord::Migration
+  def change
+    create_table :zipcode_rates do |t|
+      t.string :zip_code
+      t.string :mha_code
+      t.string :string
```
Comment (jkassemi) [inline db/migrate/20180522142853_create_zipcode_rates.rb]: What's this column for?

File: db/migrate/20180522142853_create_zipcode_rates.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+      t.string :mha_name
+      t.string :mha_rate
```
Comment (jkassemi) [inline db/migrate/20180522142853_create_zipcode_rates.rb]: Should `mha_rate` and `mha_rate_grandfathered` be strings?

File: db/migrate/20180522142853_create_zipcode_rates.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+class CreateZipcodeRates < ActiveRecord::Migration
+  def change
+    create_table :zipcode_rates do |t|
+      t.string :zip_code
```
Comment (jkassemi) [inline db/migrate/20180522142853_create_zipcode_rates.rb]: Does this need a `version` in order to track entries based on the source file that was uploaded?

File: db/migrate/20180522142853_create_zipcode_rates.rb (hunk 1 above)
Comment (maneshkommidi) [reply db/migrate/20180522142853_create_zipcode_rates.rb]: Sorry @jkassemi will correct it

File: db/migrate/20180522142853_create_zipcode_rates.rb (hunk 3 above)
Comment (maneshkommidi) [reply db/migrate/20180522142853_create_zipcode_rates.rb]: I do not think for this model we need version.

Comment (jkassemi) [reply db/migrate/20180522142853_create_zipcode_rates.rb]: FYI I may be misguided on ^.

Comment (maneshkommidi) [reply db/migrate/20180522142853_create_zipcode_rates.rb]: I don't think we will need version for this model

**Review by jkassemi — APPROVED** [review]
Nothing major pops out, so 👍 - we can iterate as necessary prior to this getting hooked up for production use.

**Pull request merged**